CONTRIBUTING.md

```diff
@@ -1,38 +1,46 @@
 # Contributing
 
-## Guidelines
+## Guidelines
+
 :octocat: It is recommended to have an issue open for any work you take on and intend to submit as a pull request - it helps core members review your concept and direction early and is a good way to discuss what you're planning to do. If there is no issue tied to the Pull Request, please include a detailed description in the PR.
 
 :100: We are aiming for 100% code coverage. Please keep this in mind when you're submitting new code.
 
 :ok_hand: We use [PSR standard](http://www.php-fig.org/psr/) to keep our code sanitized and easy to follow.
 
-## Environment requirements
+## Environment requirements
+
 `PHP >= 5.4` with `mbstring` extension
 ```bash
 ; php.ini
 extension=php_mbstring.dll
 ```
 
-## Development setup
+## Development setup
+
 1. Fork our repository
 2. Clone your forked repository `git clone git@github.com:<your namespace>/bottomline.git`
 2. Install development dependencies `make install-dev`
 3. Run tests `make test`
 
-## Development checklist
+## Development checklist
+
+- Add or update phpDocs for the new function with a **Usage** and **Result** section
 - Always add tests for the code that you write, including edge cases
 - Place the new functions where they belong (collections, arrays, utilities, etc.)
 - Add `README.md` doc entry for the functions
-- Add function signatures to `__/load.php`
-- Update the benchmark `bench.php`, this helps us to validate the performance of the library
+- Execute the `phpDocGen.php` script to automatically build an updated `load.php`

```

Comment:
Will see if this could be unversioned and be the responsibility of the one pushing the build to Composer.

But KISS - and also it may be better to have it anyway directly in the GitHub repository (for the one using the GitHub repo directly in their `composer.json`)